starter-code/bin/seed.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+const User = require('../public/models/user');
+
+const dbName = 'Roles';
+mongoose.connect(`mongodb://localhost/${dbName}`);
+
+const users = [
+  {
+    username : 'rick',
+    password : '123',
```

Comment:
Hay que hacer un hash de la password porque al momento de log in vas a comparar una password que lleva hash con una normal y e login non funciona